src/DSP.jl

```diff
@@ -7,43 +7,20 @@ module DSP
 macro importffts()
     quote
         using AbstractFFTs
-        if VERSION >= v"0.7.0-DEV.602"
-            using FFTW
-        end
+        using FFTW
```

Comment:
I think we can get rid of this macro and replace it's uses with `using AbstractFFTs` and/or `using FFTW`.